src/js/views/patients/worklist/worklist_views.js

```diff
@@ -222,6 +223,43 @@ const EmptyFindInListView = View.extend({
   `,
 });
 
+const CountView = View.extend({
+  template: hbs`
+    {{#if hasReachedMaximum}}
+      <div>
+        {{#if isFindInListApplied}}
+          {{#if isFlowList}}
+            {{formatMessage (intlGet "patients.worklist.worklistViews.countView.maximumFlowsCountNarrowed") itemCount=count maximumCount=maximumCount}}
+          {{else}}
+            {{formatMessage (intlGet "patients.worklist.worklistViews.countView.maximumActionsCountNarrowed") itemCount=count maximumCount=maximumCount}}
+          {{/if}}
+        {{else}}
+          {{formatMessage (intlGet "patients.worklist.worklistViews.countView.maximumListCount") maximumCount=maximumCount listTypeLabel=listTypeLabel}}
+        {{/if}}
+      </div>
+      <div>{{ @intl.patients.worklist.worklistViews.countView.narrowFilters }}</div>
+    {{else}}
+      <span class="worklist-list__count-total">
+        {{#if isFlowList}}
+          {{formatMessage (intlGet "patients.worklist.worklistViews.countView.flowsCount") itemCount=count}}
+        {{else}}
+          {{formatMessage (intlGet "patients.worklist.worklistViews.countView.actionsCount") itemCount=count}}
+        {{/if}}
+      </span>
+    {{/if}}
```

Comment:
This got a little complicated because the messaging needs to accommodate plural vs. non-plural wording (i.e. `1 Action` vs. `1 Actions`).

Reply:
Updated these to use a `getTemplate()`. Should be simpler now, but open to any other ways to simplify them more.